README.md

```diff
@@ -182,13 +182,14 @@ HTML syntax is the de facto language on the web and it's designed for building u
 
 ### Libraries / Frameworks
 - [Flux- like event controller for Riot](https://github.com/jimsparkman/RiotControl)
-- [riotux - Simple Event Controller for Riot.js](https://github.com/luisvinicius167/riotux)
+- [Mockstate - Simple Event Controller for Riot.js](https://github.com/luisvinicius167/mockstate)
```

Comment:
please remove Mockstate since it doesn't even provide demos for riot

Reply:
@GianlucaGuarini 

OK. It has been deleted.